python/addBTV.py

```diff
@@ -9,7 +9,19 @@ def update_jets_AK4(process):
     # Based on ``nanoAOD_addDeepInfo``
     # in https://github.com/cms-sw/cmssw/blob/master/PhysicsTools/NanoAOD/python/nano_cff.py
     _btagDiscriminators = [
-        'pfJetProbabilityBJetTags'
+        'pfJetProbabilityBJetTags',
+	'pfDeepCSVJetTags:probb',
+        #'pfNegativeDeepCSVJetTags:probb',
```

Comment:
Do we need all these commented discriminators? If not, just to avoid confusion, can you please remove them?

Reply:
Fair point, since they don't get stored now anyway. We'll be adding the `pfNegativeDeepCSVJetTags` back in because it's used for one SF method, but that can be done later.